packages/design-system-react-native/src/components/Text/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './Text';
+export { TextVariant, TextColor } from './Text.types';
```

Comment:
I think we also want to export the props types so they can be used to extend props 
```suggestion
export { TextVariant, TextColor } from './Text.types';
export type { TextProps } from './Text.types';
```